src/components/DiscoverSentance/DiscoverSentance.jsx

```
@@ -42,9 +42,9 @@ const SpeakSentenceComponent = () => {
 
   const callConfettiAndPlay = () => {
     let audio = new Audio(levelCompleteAudioSrc);
-
     audio.play();
     callConfetti();
+    window.telemetry?.syncEvents && window.telemetry.syncEvents();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)